apps/bfd-pipeline/bfd-pipeline-rif-load/src/test/java/gov/cms/bfd/pipeline/rif/load/RifLoaderIT.java

```diff
@@ -131,6 +131,21 @@ public void loadSyntheticData() {
     loadSample(dataSource, StaticRifResourceGroup.SYNTHETIC_DATA);
   }
 
+  /**
+   * Runs {@link gov.cms.bfd.pipeline.rif.load.RifLoader} against the {@link
+   * StaticRifResourceGroup#LOCAL_SYNTHETIC} data.
+   */
+  @Test
+  public void loadLocalSyntheticData() {
+    /*Assume.assumeTrue(
+    String.format(
+        "Not enough memory for this test (%s bytes max). Run with '-Xmx5g' or more.",
+        Runtime.getRuntime().maxMemory()),
+    Runtime.getRuntime().maxMemory() >= 4500000000L); */
```

Comment:
Can you add a comment explaining why this code is commented out, and when one might _un_ comment it?